src/api/client.go

```diff
@@ -272,11 +272,13 @@ func (c *Client) PostJSONV2(endpoint string, reqObj, respObj interface{}) (bool,
 		return false, rspErr
 	}
 
-	decoder = json.NewDecoder(bytes.NewReader(wrapObj.Data))
-	decoder.DisallowUnknownFields()
+	if respObj != struct{}{} {
```

Comment:
What is this check for

Reply:
![image](https://user-images.githubusercontent.com/8619106/50539358-5f6d0380-0ba5-11e9-9dee-282d32a66fa0.png)

I think its this

Reply:
The advantage is nullified by avoiding `DisallowUnknownFields()` when the struct is a `struct{}{}`